iroha-transfer-asset/step3.md

```diff
@@ -1,31 +1,28 @@
-Let's create another file:
+Now as Alice has her account and a private key, she can send commands to the system. 
```

Comment:
This is a bit misleading. People would think that the previous script creates Alice's account. In fact, we just configure a library to use that particular private key of Alice that was pre-configured inside a Genesis block. Shall we mention this somewhere?

Reply:
Fixed (hopefully)